ci/jenkins/pipelines/prs/helpers/pr_manager/pr_manager.py

```diff
@@ -50,23 +50,26 @@ def check_pr(args):
 
 
 def filter_pr(args):
-    g = Github(GITHUB_TOKEN, per_page=1000)
-    repo = g.get_repo(GITHUB_REPO)
+    if CHANGE_ID:
+        g = Github(GITHUB_TOKEN, per_page=1000)
+        repo = g.get_repo(GITHUB_REPO)
 
-    pull = repo.get_pull(int(CHANGE_ID))
-    changed_files = pull.get_files()
-    files_list = []
+        pull = repo.get_pull(CHANGE_ID)
+        changed_files = pull.get_files()
+        files_list = []
 
-    # change paginated list to normal list
-    for f in changed_files:
-        files_list.append(f.filename)
+        # change paginated list to normal list
+        for f in changed_files:
+            files_list.append(f.filename)
 
-    if any([s for s in files_list if args.filename in s]):
-        msg = "contains"
-    else:
-        msg = "does not contain"
+        if any([s for s in files_list if args.filename in s]):
+            msg = "contains"
+        else:
+            msg = "does not contain"
 
-    print(f"Pull Request {GITHUB_REPO}#{CHANGE_ID} {msg} changes for {args.filename}")
+        print(f"Pull Request {GITHUB_REPO}#{CHANGE_ID} {msg} changes for {args.filename}")
+    else:
+        print(f'No CHANGE_ID was set assuming this is not a PR. Skipping filters...')
```

Comment:
No need to have `f` if no string interpolation.

Reply:
Thx, python3 expect!